packages/lockfile-lint/README.md

```diff
@@ -78,6 +78,19 @@ lockfile-lint --path yarn.lock --allowed-hosts yarn github.com --validate-https
 | `--validate-checksum`, `-c`  | check that all resources include a checksum                                                                                                                                                                                                                                                | ❌ PRs welcome |
 | `--validate-integrity`, `-i` | check that all resources include an integrity field                                                                                                                                                                                                                                        | ❌ PRs welcome |
 
+# File-Based Configuration
+
+Lockfile-lint uses [cosmiconfig]() for configuration file support. This means you can configure the above options via (in order of precedence):
```

Comment:
probably want to add the npmjs link for`cosmiconfig`

Reply:
😁 oops, sorry! I _meant_ to...

Reply:
Fixed in 3b61e37